tests/run/t3346e.flags

```diff
@@ -0,0 +1 @@
+-language:Scala2
```

Comment:
I was compiling a community build library, an error message led me to the test. It actually works with current Dotty, it's good to move things from pending.